routes/transaction.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const { postCreateTransaction } = require('../controllers/transaction');
+const postTransactionSchema = require('../schemas/transaction/postTransactionSchema');
+const validator = require('../middlewares/validator');
+const router = express.Router();
+
+router.post('/', validator(postTransactionSchema), postCreateTransaction);
```

Comment:
You must create a folder called transactions and inside place the js of the route.

![Captura de Pantalla 2022-11-03 a la(s) 8 04 19 p m](https://user-images.githubusercontent.com/84211273/199863122-b0f818f1-525c-4a5d-afd2-483ac2c507e5.png)